superset-frontend/src/explore/components/controls/SelectControl.jsx

```diff
@@ -102,10 +102,9 @@ export default class SelectControl extends React.PureComponent {
   // Beware: This is acting like an on-click instead of an on-change
   // (firing every time user chooses vs firing only if a new option is chosen).
   onChange(opt) {
-    let optionValue = null;
+    let optionValue = this.props.multi ? [] : null;
```

Comment:
One more thing - I might be missing something here, but I don't quite get why this can't always be initialized to `[]`. @maloun96 can you check if that works? I'd much rather not have a ternary here that can be avoided.

Reply:
@villebro  in case that select has only 1 value it should be initialized with null, only if it can have multiple select that it should be []